app_flutter/test/widgets/task_overlay_test.dart

```diff
@@ -253,17 +249,16 @@ void main() {
         ),
       ),
     );
+    await tester.pump();
 
     expect(find.text(expectedTaskInfoString), findsNothing);
-    // TODO(chillers): Fix flakiness of precacheTaskIcons. https://github.com/flutter/flutter/issues/58833
-    // await expectLater(find.byType(MaterialApp), matchesGoldenFile('task_overlay_test.nondevicelab_closed.png'));
+    await expectLater(find.byType(MaterialApp), matchesGoldenFile('task_overlay_test.nondevicelab_closed.png'));
```

Comment:
Did a quick test locally, after adding `await tester.pump()`, it seems to still throw the diff error.
Anything else to change to get it passed?

Reply:
Can you post the log? These tests should only be run on Linux. I have a followup PR that will skip these tests on non-linux platforms.

Reply:
See https://github.com/flutter/flutter/issues/60720 for running golden tests only on linux.

Reply:
Oh, i see. I ran it in Mac.